connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -209,6 +209,21 @@ message WriteStreamOperationStart {
     string path = 11;
     string table_name = 12;
   }
+
+  StreamingForeachWriter foreach = 13;
```

Comment:
How about renaming this `foreach_writer`. That matches the `Writer` suffix used in message name. Leaving it as just `foreach` looks quite generic.